test/framework/deployments/externalservice/kubernetes.go

```diff
@@ -76,3 +76,7 @@ func (k *k8SDeployment) Delete(cluster framework.Cluster) error {
 func (k *k8SDeployment) GetExternalAppAddress() string {
 	return k.ip
 }
+
+func (k *k8SDeployment) GetCert() string {
+	return ""
```

Comment:
why is it empty on K8S?

Reply:
It is actually not implemented there. Will just panic.

Reply:
Ok, but why? Is it because we don't have this test on Kubernetes? If so, please add a comment why we don't have a test on Kubernetes